.github/workflows/update-subtree.yml

```diff
@@ -77,6 +74,24 @@ jobs:
       env:
         GH_TOKEN: ${{ github.token }}
 
+    - name: Checkout splitsh-lite
+      if: ${{ env.SUBTREE_PR_EXISTS == 'no' }}
+      uses: actions/checkout@v4
+      with:
+        repository: splitsh/lite
+        path: splitsh-lite
+
+    - name: Build splitsh-lite
+      if: ${{ env.SUBTREE_PR_EXISTS == 'no' }}
+      run: |
+        cd splitsh-lite
+        sudo apt-get install -y libgit2-dev
+        sed -i 's#github.com/libgit2/git2go#github.com/jeffwelling/git2go#' go.mod splitter/*
+        sed -i -e 's/v34/v37/g' go.mod splitter/*.go
+        sed -i 's/v37.0.0/v37.0.4/' go.mod
+        go mod tidy
```

Comment:
Is `go` guaranteed to be available, or should we add a step that sets it up?

Reply:
Fair point, I'll play it safe and will add it to the `apt-get install` line.